### ISSUE TITLE
Crash when editing lib.xeto depends

### ISSUE THREAD

**Issue by briansfrank** [Novo-Int/xeto-vscode-extension#42]
I've had the lang server crash a lot when I'm editing my lib.xeto depends.  I haven't figure out the exact sequence to reproduce, but its creating new empty { lib: "" } definitions, removing/adding curly braces.  Trace is below.  If you can't reproduce it I try to see if I can come up with steps...

node_modules/vscode-jsonrpc/lib/common/events.js:117:36)
	at closeHandler (/Users/brian/.vscode/extensions/novo-studio.xeto-extension-0.1.26/client/node_modules/vscode-jsonrpc/lib/common/connection.js:256:26)
	at CallbackList.invoke (/Users/brian/.vscode/extensions/novo-studio.xeto-extension-0.1.26/client/node_modules/vscode-jsonrpc/lib/common/events.js:55:39)
	at Emitter.fire (/Users/brian/.vscode/extensions/novo-studio.xeto-extension-0.1.26/client/node_modules/vscode-jsonrpc/lib/common/events.js:117:36)
	at IPCMessageReader.fireClose (/Users/brian/.vscode/extensions/novo-studio.xeto-extension-0.1.26/client/node_modules/vscode-jsonrpc/lib/common/messageReader.js:40:27)
	at ChildProcess.<anonymous> (/Users/brian/.vscode/extensions/novo-studio.xeto-extension-0.1.26/client/node_modules/vscode-jsonrpc/lib/node/main.js:33:45)
	at ChildProcess.emit (node:events:525:35)
	at maybeClose (node:internal/child_process:1112:16)
	at ChildProcess._handle.onexit (node:internal/child_process:304:5)

**Comment by rracariu** [comment]
Looks that there is a undefined check that blows up:
```
.vscode/extensions/novo-studio.xeto-extension-0.1.26/server/out/parseDocument.js:55
                deps.push(protoDeps[key].children.lib.type);
                                                      ^

TypeError: Cannot read properties of undefined (reading 'type')
    at vscode/extensions/novo-studio.xeto-extension-0.1.26/server/out/parseDocument.js:55:55
```
Just adding an empty  block `{ }` would trigger this.

**Comment by hades2510** [comment]
this should be fixed in 0.1.27

**Comment by briansfrank** [comment]
I tested it out with 0.1.27 and looks like its fixed to me, thanks
